packages/site-server/src/catalog/routes/element/element-route.ts

```diff
@@ -78,22 +82,26 @@ export const handleElementRoute = async (
     throw new Error('Internal error');
   }
 
-  context.body = Readable.from(renderPage({
-    title: `${packageName}/${elementName}`,
-    scripts: [
-      '/js/hydrate.js',
-      '/js/element.js'
-    ],
-    content: render(
-      renderElement({
-        packageName: packageName,
-        elementName: elementName,
-        declarationReference: customElement.declaration,
-        customElementExport: customElement.export,
-        manifest: customElementsManifest,
-      })
-    ),
-  }));
+  const elementData = {
+    packageName: packageName,
+    elementName: elementName,
+    declarationReference: customElement.declaration,
+    customElementExport: customElement.customElementExport,
+    manifest: customElementsManifest,
+  };
+
   context.type = 'html';
   context.status = 200;
+  context.body = Readable.from(
+    renderPage({
+      title: `${packageName}/${elementName}`,
+      scripts: ['/js/hydrate.js', '/js/element.js'],
+      // TODO (justinfagnani): If we want to hydrate, we need to serialize the
+      // initial data and embed in the response
+      content: render(
+        html`<wco-element-page .elementData=${elementData}></wco-element-page>`,
+        {deferHydration: true}
```

Comment:
@augustjk @kevinpschaaf @aomarks SSR DX issue here: the error I got before adding `deferHydration: true` was that an unexpected TemplateResult was encountered on the client (due to a conditional in the element when no data was present). That would have been hard to track down for a non-team-member.